modules/publicize/publicize-jetpack.php

```diff
@@ -447,7 +447,11 @@ function test_connection( $service_name, $connection ) {
 	 * Save a flag locally to indicate that this post has already been Publicized via the selected
 	 * connections.
 	 */
-	function save_publicized( $post_ID, $post, $update ) {
+	function save_publicized( $post_ID = null, $post = null, $update = null ) {
+		if ( is_null( $post_ID ) || is_null( $post ) || is_null( $update ) ) {
```

Comment:
Since we're not using `$update` in this function, and relying on the publicize meta being set, should we care if `$update` is null?